rust/operator-binary/src/controller.rs

```diff
@@ -925,6 +926,12 @@ fn build_rolegroup_statefulset(
             "config",
             "log",
             merged_config.logging.containers.get(&Container::Vector),
+            ResourceRequirementsBuilder::new()
```

Comment:
I'd say this deserves a changelog callout as well

Reply:
Agreed! Let's way till we decided if we are going to merge podOverrides or the whole resource stuff first

Reply:
Is it viable to split this out into a separate PR?

Reply:
Some context: This is a breaking change in operator-rs (the resources are now required), which we decided to release together with the resource PR. So I *need* to explicitly specify the resources.
However, after thinking about this, I would simply re-use the default we had previously in operator-rs, to *not* change the behavior in any way. So this PR can be completely independent of the resources PR that will follow.

WDYT?

Reply:
Nevermind, we never had resources set for the vector container, so we need to decide on some value now

Reply:
Updated the resources to the value we agreed this morning

Reply:
Who's "we" in this context? Especially the inline comment is pretty useless and could do with a reference back to that discussion.

Reply:
Maxi, Razvan, Sasch, Jim, Malte and me (I might missed someone) in the meeting this morning at 09. Anyway, I know this was going to happen when we stuff everything in a single operator-rs release :upside_down_face: 
Let's wait till the resource PRs have been merged before continuing with podOverrides